pom.xml

```diff
@@ -1132,5 +1078,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <modules>
+                <module>oshi-demo</module>
+                <module>oshi-dist</module>
+            </modules>
+            <properties>
+                <!-- Animal Sniffer for jdk 8 -->
+                <animal-sniffer-maven-plugin.version>1.20</animal-sniffer-maven-plugin.version>
+
+                <!-- Signature Settings -->
+                <signature.artifact>java18</signature.artifact>
+                <signature.version>1.0</signature.version>
+            </properties>
+            <build>
+                <pluginManagement>
```

Comment:
what is the need or benefit for pluginManagement in a profile?  If we're changing the profile to "opt in" instead of "opt out" why can't it just be all in one section?